django/contrib/gis/geos/geometry.py

```diff
@@ -252,9 +252,16 @@ def empty(self):
 
     @property
     def hasz(self):
-        "Return whether the geometry has a 3D dimension."
+        "Return whether the geometry has a Z dimension."
         return capi.geos_hasz(self.ptr)
 
+    @property
+    def hasm(self):
```

Comment:
I think this is missing documentation (see `.. attribute:: GEOSGeometry.hasz`).
That would also need `.. versionadded:: 6.0` and we need a small release note in the 6.0 release notes :+1: